frontend/archive/src/main/java/org/pytorch/serve/archive/model/ModelConfig.java

```diff
@@ -402,6 +409,13 @@ public static TorchRun build(Map<?, ?> torchRunMap) {
                                     logger.warn("Invalid torchrun.node-rank:{}, reset to 0", v);
                                 }
                                 break;
+                            case "OMP_NUMBER_THREADS":
```

Comment:
Is the value getting picked from the model-config.yaml? Please add steps in the documentation as well for how user's can set this value

Reply:
The default value is 1, which is auto set in the code. The value can be modified in model-config.yaml.